unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Animator/DCLAnimator.cs

```diff
@@ -47,8 +47,8 @@ public class DCLAnimationState
 
         public override IEnumerator ApplyChanges(BaseModel model)
         {
-            entity.OnShapeUpdated -= OnComponentUpdated;
-            entity.OnShapeUpdated += OnComponentUpdated;
+            entity.OnShapeLoaded -= OnComponentUpdated;
```

Comment:
```suggestion
            entity.OnShapeLoaded -= OnShapeLoaded;
```
I think we can safely rename the method to `OnShapeLoaded`